hw/top_earlgrey/ip/sensor_ctrl/lint/sensor_ctrl.waiver

```diff
@@ -0,0 +1,14 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+  #
+# waiver file for sensor_ctrl
+
+waive -rules HIER_NET_NOT_READ -location {sensor_ctrl_reg_top.sv} -regexp {reg_wdata} \
+      -comment "No registers are 32 bits"
+
+waive -rules HIER_NET_NOT_READ -location {sensor_ctrl_reg_top.sv} -regexp {reg_be} \
+      -comment "No registers are 32 bits"
+
+waive -rules NOT_READ -location {sensor_ctrl_reg_top.sv} -regexp {reg_wdata} \
+      -comment "No registers are 32 bits"
```

Comment:
nit: looks like this file is missing a new line...

Reply:
i've always wondered about that. Is this something that we have to have?
why is not having a newline bad?

Reply:
Good question... for me this somehow goes into the same bucket of "good practices" like deleting trailing whitespace from a file :).

It may not matter much for this particular waiver file, but I thought this "good practice" is to avoid surprises in parsing scripts or file concatenation operations, which can sometimes get messed up by a file not having an endline character.